src/main/java/duke/databse/Database.java

```diff
@@ -0,0 +1,215 @@
+package duke.databse;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.core.UI;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+
+/**
+ * A class to CRUD file that stores task
+ */
+public class Database {
+
+    private File file;
+    private UI ui;
+
+    /**
+     * Constructor for database
+     * @param filePath
+     */
+    public Database(String filePath) {
+        try {
+            file = new File(filePath);
+            ui = new UI();
+            if (file.createNewFile()) {
+                System.out.println(ui.creating_file_message);
+            } else {
+                System.out.println(ui.reading_file_message);
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred in creating or opening file.");
+        }
+    }
+
+
+    /**
+     * read data from file
+     * @return
+     */
+    public ArrayList<Task> getData() {
+        ArrayList<Task> objectsList = new ArrayList<>();
+        try {
+            Scanner myReader = new Scanner(file);
+            while (myReader.hasNextLine()) {
+                String data = myReader.nextLine();
+                objectsList.add(parseData(data));
+            }
+        } catch (IOException e) {
+            System.out.println(ui.wrong_message);
+        }
+        return objectsList;
+    }
+
+    /**
+     * parse data from string to task
+     * @param data
+     * @return
+     */
+    public Task parseData(String data) {
+        String[] s = data.split(" ");
+
+        boolean isDone;
+        if (s[s.length - 1].equals("false")) {
+            isDone = false;
+        } else {
+            isDone = true;
+        }
+        switch (s[0]) {
```

Comment:
Could consider assigning s[0] to a variable before using it in this switch statement? Understand your intention but would be more readable if the variable to compare is a bit more explicit.